docs/coding-style.md

```diff
@@ -98,6 +98,12 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
   * especially when each branch ends with a `return` statement.
 * Explicit `else` is not used after input validation.
 * Each compiler directive outside method body (namely `#if`/`#endif`) should be preceded and followed by an empty line.
+* When there are multiple conditions to be met before the core logic of a method can be executed, we have a preference in structuring these conditions:
```

Comment:
I would move this whole block one line higher - to have the if/else/logic related topics together. And the `#directives`  as a last point, as it is a different topic.

Reply:
This sentence is quite wordy, I'd prefer to simplify the introduction to stay to the point
```suggestion
* For multiple conditions before the core method logic:
```
On the next paragraph, we'll need to continue with `chain` instead of `chained`.